pkg/evs/controllerserver.go

```diff
@@ -517,6 +488,7 @@ func querySnapshotPageList(credentials *config.CloudCredentials, req *csi.ListSn
 	for _, element := range pageList.Snapshots {
 		responses = append(responses, generateListSnapshotsResponseEntry(&element))
 	}
+	log.Infof("Successful query snapshot list. detail: %v", protosanitizer.StripSecrets(responses))
 	return &csi.ListSnapshotsResponse{Entries: responses}, nil
```

Comment:
`NextToken` is missing

Reply:
v2 API not match cinder API, v2 API does not have this field

Reply:
adjusted